View/AggregatedStageDamage/CalculatedStageDamageControl.xaml.cs

```diff
@@ -19,7 +19,57 @@ private void calculate_btn_Click(object sender, RoutedEventArgs e)
         {
             if (DataContext is CalculatedStageDamageVM vm)
             {
-                vm.CalculateCurves();
+                vm.ComputeCurves();
+            }
+        }
+
+        /// <summary>
+        /// This is to stretch the listview columns to fill the space as the user stretches the window.
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="e"></param>
+        private void ListView_SizeChanged(object sender, SizeChangedEventArgs e)
+        {
+            ListView listView = sender as ListView;
+            GridView gView = listView.View as GridView;
+            
+            // take into account vertical scrollbar
+            var workingWidth = listView.ActualWidth - SystemParameters.VerticalScrollBarWidth - 5; 
+            var col1 = 0.33;
+            var col2 = 0.33;
```

Comment:
if they're all .33, can we just use one variable?

Reply:
done